podcasts/en.lproj/Localizable.strings

```diff
@@ -55,6 +55,15 @@
 /* A common string used throughout the app. An accessibility label to direct the user to sort and option menus. */
 "accessibility_sort_and_options" = "Sort and Options";
 
+/* Accessibility label for episode playback progress scrubber */
+"accessibility_playback_progress" = "%1$@ of %2$@";
```

Comment:
In theory, we shouldn't add new strings when targeting release branches. We won't have time to get this merged, picked up for translation, and then translated.

Anyway, we do have a few exceptions and this is one so let's go ahead and merge it. :)